content/200-concepts/150-data-platform/01-about-platform/100-platform-limits-and-status.mdx

```diff
@@ -0,0 +1,88 @@
+| --------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------- |
+| Maximum number of projects on the **Free** plan per account<br /><br /><br />**Note**<br /><br /> The number does not include projects on the **Early Adopter** plan. | 10             |
+| Maximum number of projects per account                                                                                                                                | 100            |
+| Maximum number of collaborators per project                                                                                                                           | 100            |
+| Maximum number of environments per project                                                                                                                            | 100            |
+| Maximum number of connection strings per environment                                                                                                                  | 100            |
+| Response timeout in Data Browser                                                                                                                                      | 60s            |
+| Response timeout in Query Console                                                                                                                                     | 60s            |
+| Maximum number of rows fetched in Data Browser or Query Console                                                                                                       | Not applicable |
+
+<Admonition>
+
+**Extending general limits**<br /><br />
+Contact Support if you need to have any of the existing limits lifted for your project.
+
+</Admonition>
+
+## Concurrency limits of the Prisma Data Proxy
+
+The concurrency limits of the Prisma Data Proxy define the maximum number of proxy instances that can run simultaneously for a single project. By default, at least one proxy instance is started for a single project environment and additional instances are started when an increase in traffic demands that.
+
+In addition, more proxy instances start for a project environment under these circumstances:
+
+- multiple Prisma Clients connect that use a different version of the Prisma Client
+- multiple Prisma Clients connect that use a different version of the Prisma schema
+
+| Concurrency limits                                                                                         | Number                            |
+| ---------------------------------------------------------------------------------------------------------- | --------------------------------- |
+| Connection pool per proxy instance                                                                         | **Default**: 60<br />**Max**: 120 |
```

Comment:
```suggestion
```

This should be removed. The required change is not landing yet.

Reply:
Removed.